lib/watir/elements/svg_elements.rb

```diff
@@ -634,15 +580,15 @@ def text_paths(*args)
     end
     Watir.tag_to_class[:textPath] = TextPath
 
-    # @return [TSpan]
+    # @return [Text]
```

Comment:
I can't figure out why the generator is making this `Text` instead of `TSpan`. Everything works if I manually change it back. We can:
1. Manually change it and create issue to investigate more later
2. Add it to the ignored list (like I also ended up doing with unknown tag) 
3. Figure out how to fix the generator code

I'd like to get this code released out pretty much right away since there are a couple bug fixes that people need. So, @p0deje, if 3 isn't an option for you right away, let me know if you prefer 1 or 2 so I can do the release. Thanks.